### ISSUE TITLE
Inference on test data

### ISSUE THREAD

**Issue by abhijeet-gorai-capg** [MontaEllis/Pytorch-Medical-Segmentation#47]
How can we inference on test data without having labels?

**Comment by MontaEllis** [comment]
You can remove label code in dataloader.
